tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -348,7 +348,6 @@ def targets():
         ProtocArtifact('linux', 'x64'),
         ProtocArtifact('linux', 'x86'),
         ProtocArtifact('macos', 'x64'),
-        ProtocArtifact('macos', 'x86'),
```

Comment:
as is, this will break the build of Grpc.Tools  nuget package (run an adhoc artifact/packages/distribtest flow to see that)
two options:
- we remove the x86 protoc artifact in a separate PR (and I can help with adjusting C#).
- you also include the Grpc.Tools fix here

Reply:
Removing these should be enough to make the nuget package build pass?
https://github.com/grpc/grpc/blob/2dc98b1e4cfa433140bc1c7af0a99b718720331a/src/csharp/Grpc.Tools/Grpc.Tools.csproj#L72
https://github.com/grpc/grpc/blob/2dc98b1e4cfa433140bc1c7af0a99b718720331a/src/csharp/Grpc.Tools/Grpc.Tools.csproj#L65

Reply:
Thanks! I included this in this PR.